templates/taxi/driver_list.html

```diff
@@ -7,7 +7,10 @@ <h1>Driver list</h1>
       {% for driver in driver_list%}
         <li>
           <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
-          ({{ driver.first_name }} {{ driver.last_name }})
+          {% if driver.id == user.id %}
+          (Me)
+          {% else  %} ({{ driver.first_name }} {{ driver.last_name }})
+          {% endif %}
```

Comment:
Due to the task, we need to add (Me) if it is the current user. Noting said to remove first_name and last name from text

Reply:
You didn’t provide this change. You don’t need else block here